src/components/30-organisms/table-sortable/index.js

```diff
@@ -158,14 +171,43 @@ class AXATableSortable extends LitElement {
       const cleanCellRx = cellRx.replace(/<[^>]*>/g, '').trim();
       const cleanCellLx = cellLx.replace(/<[^>]*>/g, '').trim();
       let result;
-      // eslint-disable-next-line no-restricted-globals
+
+      if (
+        this.dateSortColumnIndex &&
+        this.dateColumsCustomSort.includes(index)
+      ) {
+        const cleanDateLx = new Date(this.convertDateToUSFormat(cleanCellLx));
+        const cleanDateRx = new Date(this.convertDateToUSFormat(cleanCellRx));
+
+        if (!isNaN(cleanDateLx.getTime()) && !isNaN(cleanDateRx.getTime())) {
+          result = sortDate(cleanDateLx, cleanDateRx);
+          return sortAs === ASC ? result : ~result + 1;
```

Comment:
@NoahLiechti please dont use `~` here.

Reply:
I have it copied from an other snippet in the same component, but it makes sense to change it

Reply:
Yes my bad 🤗 Thank you

Reply:
done